README.md

```diff
@@ -586,6 +586,111 @@ crud.len('customers')
 ...
 ```
 
+### Statistics
+
+`crud` routers provide statistics on called operations.
+```lua
+-- Returns table with statistics information.
+crud.stats()
+```
+
+Statistics collect enabled by default on all routers.
+It can be disabled and re-enabled later.
+```lua
+-- Disables statistics collect and resets all collectors.
+crud.disable_stats()
+
+-- Enable statistics collect and recreates all collectors.
+crud.enable_stats()
+```
+
+If [`metrics`](https://github.com/tarantool/metrics) found,
+you can use metrics collectors to store statistics
+instead of local collectors.
+It is required to use version `0.9.0` or greater,
+otherwise local collectors will be used.
+```
+-- Use metrics collectors.
+crud.enable_stats({ driver = 'metrics' })
+```
+By default, local collectors (`{ driver = 'local' }`)
+are used. Metrics collectors are much sophisticated and
+would show execution time quantile with aging for calls.
+Be wary that computing quantiles may affect overall
```

Comment:
Should we enable metrics by default in this case? Some users could consider this as regression.

Reply:
Computing quantiles is not enabled by default (metrics integration is enabled manually). But now I'm researching performance issues and it could be my assumption is incorrect and it was a deal of `--coverage` flag. I'll update it later

Reply:
Reworked both performance tests and registry.
```
test_insert_with_stats_disabled: requests 693431, rps 23114, errors 0
test_insert_with_local_stats: requests 676918, rps 22563, errors 0
test_insert_with_metrics_stats: requests 612714, rps 20423, errors 0

test_select_with_stats_disabled: requests 238300, rps 7943, errors 0
test_select_with_local_stats: requests 240500, rps 8016, errors 0
test_select_with_metrics_stats: requests 219180, rps 7306, errors 0
```
in CI (similar results on local machine). There is a performance drop in using metrics driver (now used by default if metric available), but it is not crucial. I'm not sure what we should do about this.

Reply:
https://github.com/tarantool/crud/pull/244#discussion_r770261344

Reply:
https://github.com/tarantool/crud/pull/244#issuecomment-996749316